README.md

```diff
@@ -1,19 +1,27 @@
-Cover-to-Cobertura Conversion Tool 
-==================================
+# Cover-to-Cobertura Conversion Tool 
+
+[![Hex.pm version](https://img.shields.io/hexpm/v/rebar_covertool.svg?style=flat)](https://hex.pm/packages/rebar_covertool)
 
 A simple tool to convert exported Erlang `cover` data sets into Cobertura XML
 reports. The report could be then feed to the Jenkins Cobertura plug-in.
 
-Usage
------
+## Plugin rename notice
+
+This plugin is currently known on [Hex](https://hex.pm) as [`rebar_covertool`](https://hex.pm/packages/rebar_covertool), although the repository and application have always been called `covertool`. That was originally because the rebar2 plugin was built into the `rebar_covertool` module instead of the `covertool` module. We will imminently introduce the package on Hex as [`covertool`](https://hex.pm/packages/covertool) 2.0.0, and the [`rebar_covertool`](https://hex.pm/packages/rebar_covertool) package will be **retired**.
+
+As of `covertool` 1.3.0, the `covertool` module now contains callbacks for all supported plugin formats (rebar 2, rebar3, and mix). Going forward, referencing `covertool` as `rebar_covertool` in plugins is **deprecated**. Use `covertool` instead (see [usage](#usage) below).
+
+This change was made to make the applications more straightforward to integrate with. See [#43](https://github.com/covertool/covertool/issues/42) for more information.
```

Comment:
Was this supposed to be `See [#42] (https://github.com/covertool/covertool/issues/42)...` ?

Reply:
Yes. Fixed, thanks.